src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaTeachingAssignmentService.java

```diff
@@ -50,7 +50,7 @@ public TeachingAssignment save(TeachingAssignment teachingAssignment) {
 	@Override
 	@Transactional
 	public void delete(Long id) {
-		teachingAssignmentRepository.deleteById(id);
+		teachingAssignmentRepository.delete(id);
```

Comment:
do we still need the `deleteById` method in `TeachingAssignmentRepository.java`?

Reply:
Identical to `delete(id)`? Nomenclature issue?

Reply:
Not identical but no longer used.  Ill delete it.  delete is an out of the box function but deletebyId was custom.